src/actionTypes.js

```diff
@@ -0,0 +1,5 @@
+const ADD = 'ADD';
+const REMOVE = 'REMOVE';
+const CHECK = 'CHECK';
```

Comment:
- [ ] Great job setting up the action types, but there is an issue we need to fix. According to the project requirements, which is a best practice, we are supposed to follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux), something like so(`just example`); `const ADD_BOOK = bookstore/books/ADD_BOOK`, `const REMOVE_BOOK = bookstore/books/REMOVE_BOOK`, `const CHECK_STATUS = bookstore/categories/CHECK_STATUS`.  Please let's fix that.

Reply:
I missed the requirement. Thanks for pointing it out! 😉